src/components/AvatarCropModal/ImageCropView.js

```diff
@@ -90,7 +90,7 @@ function ImageCropView(props) {
                 <View style={[containerStyle, styles.l0, styles.b0, styles.pAbsolute]}>
                     <Icon
                         src={props.maskImage}
-                        fill={theme.icon}
+                        fill={theme.iconReversed}
```

Comment:
any reason for keeping this value remained here only? I see all other `fill` values are removed

Reply:
For applying maskImage based on theme we have kept that, comment [ref](https://github.com/Expensify/App/pull/33607#issuecomment-1877418386)